package.json

```diff
@@ -7,11 +7,12 @@
         "build:webpack": "mix --production",
         "build:eleventy": "cross-env NODE_ENV=production eleventy",
         "clean": "rimraf dist",
+        "dev": "cross-env NODE_ENV=development eleventy --serve",
         "lint": "fluid-lint-all",
         "release": "standard-version -t ''",
         "start": "npm-run-all -l clean -p start:*",
         "start:webpack": "mix watch",
-        "start:eleventy": "cross-env NODE_ENV=development eleventy --serve"
+        "start:eleventy": "concurrently \"npm run dev\" \"netlify-cms-proxy-server\""
```

Comment:
Is there a need for both concurrently and npm-run-all? Would it be possible to just use one of these, or do they provide unique features for specific cases?

Reply:
It looks like npm-run-all with the parallel flag (or [run-p](https://github.com/mysticatea/npm-run-all/blob/master/docs/run-p.md)) can handle parallelized commands in the same way that concurrently does. I've removed concurrently in favour of run-p in 3b17bb6.